mlflow/utils/requirements_utils.py

```diff
@@ -222,7 +222,7 @@ def _get_installed_version(package, module=None):
         # 1.9.0+cu102
         # $ python -c "import importlib_metadata; print(importlib_metadata.version('torch'))"
         # 1.9.0
-        version = __import__(module or package).__version__
+        version = __import__(module or _PACKAGES_TO_MODULES.get(package) or package).__version__
```

Comment:
Do we need this change? Can we just call `_get_installed_version(package, _PACKAGES_TO_MODULES.get(package))`?